src/text-highlighter/text-highlighter.test.tsx

```diff
@@ -0,0 +1,14 @@
+import { render } from '@config/testing';
+import * as React from 'react';
+
+import { TextHighLighter } from './text-highlighter';
+
+describe('Dialog', () => {
+    it('should render with highlightedText', () => {
+        const content = 'Message admin';
+        const { getByDataQa } = render(<TextHighLighter content={content} highlightedTexts={['admin']} />);
+        const highlightedText = getByDataQa('highlighted-admin');
+        expect(highlightedText).toBeTruthy();
+        expect(highlightedText.style.fontWeight).toBe('bold');
```

Comment:
As mentioned in the implementation: hard coded style is not very flexible. Classes would also be easier to test

Reply:
fixed